pages/samvera/manager_guide/hyrax_2.1/collections-2.1.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Collections"
+permalink: collections-2.1.html
+folder: samvera/manager_guide/collections-2.1.md
+sidebar: home_sidebar
+a-z: ['Collections - Repository Administration']
+keywords: Best Practices, managers, repo mangers, hyrax administration
+tags: [user_resources]
+categories: How to use the Administration panel in hyrax
+version:
+  id: 'hyrax_2.1-stable'
+  versions:  
+  - label: 'Hyrax 2.0'
+    link:  'collections-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'collections-2.1.html'
+    selected: 'true'
+---
+Hyrax 2.1 comes with a suite of new options for creating and configuring collection types to satisfy a number of use cases for cultural heritage and research institutions. This page focuses on the basics of an end user creating a collection. For a detailed overview of collection types and configurations, please visit the [Collections and Admin Sets](http://samvera.github.io/collection-overview.html) section of this website.
+
+## Create a User Collection
+
+Login to the repository, navigate to the Dashboard, click on Collections in the sidebar:
+
+![Collections Dashboard](/images/screenshots/collections-2.1.png)
+
+Click on the `New Collection` button at the top of the screen. If the repository contains more than one collection type, then a dialog box will pop up, giving you the option to select a collection type for the new collection.
```

Comment:
add "Some collection types are not available to all users of the system. Users will only see choices for collection types they have permissions to create" ???

Reply:
done